### DISCUSSION TITLE
Error 404 when requesting latest observation for valid stations using /stations/{stationId}/observations/latest

### DISCUSSION THREAD

**Discussion by rosteve** [weather-gov/api#253] [Q&A]
**Describe the bug**
Is it expected to get error 404 when requesting for latest obeservations for a valid station?
Ideally, the result should be an empty list but no returning error 404 as its currently returning.

**To Reproduce**
To reproduce, use this API endpoint for SIOUX CENTER (COOPSIXI4): 
https://api.weather.gov/stations/COOPSIXI4/observations/latest

Here's the response for the above request.
{
    "correlationId": "5316761a",
    "title": "Not Found",
    "type": "https://api.weather.gov/problems/NotFound",
    "status": 404,
    "detail": "Not Found",
    "instance": "https://api.weather.gov/requests/5316761a"
}

**Expected behavior**
It should return an empty list and not error 404.

**Environment**
Google Chrome Version 87.0.4280.88 (Official Build) (x86_64)

**Comment by scadergit** [comment]
Duplicate of #149
